concordium-std/tests/derive-serial/fail-tag-without-repr.stderr

```diff
@@ -0,0 +1,7 @@
+error: 'repr(..)' attribute must be set on the type when using 'tag' attribute
+ --> tests/derive-serial/fail-tag-without-repr.rs:5:10
+  |
+5 | #[derive(Serial)]
+  |          ^^^^^^
+  |
+  = note: this error originates in the derive macro `Serial` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
These exact error messages seems less than ideal. Any simple change in the compiler, or adding comments to the test, will invalidate them since line numbers will change.

Is that the only way to test this?

Reply:
I agree that this is not ideal, but this is the workflow that `trybuild` advocate. Changes in the compiler seems like something we would want to notice, like if something else fails before this point, but yes changing the tests will mean updating these files. However, these files are generated, so it is not a big deal to update them.

Reply:
Ok.